examples/kafka_message_consumer_group_service/kafka_message_consumer_group_service.out

```diff
@@ -0,0 +1,3 @@
+# To run this sample, navigate to the directory that contains the
+# `.bal` file, and execute the `ballerina run` command.
+$ ballerina run kafka_message_consumer_group_service.bal
```

Comment:
Shall we state that the user needs to start a Kafka server, before running these examples ?

Reply:
Added a commented in code